### PR TITLE
Display List of Items on Home page: Pull Request

### PR DESCRIPTION
**Pull request by belaymit**
**In this section:**

- Created a Home page
- created a static Comments Popup

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+              <small>5 likes</small>
+            </div>
+          </div>
+          <div class="btn-container">
+            <button type="button" class="btn btn-comments" id="btn-comments">
+              comments
+            </button>
+            <button type="button" class="btn">Reservation</button>
+          </div>
+        </div> -->
```
Comment (lojacho) [inline src/index.html]: Please don't leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      </div>
+    </section>
+    <section class="popup-section">
+      <div class="popup-container">
+        <button type="button" class="btn movie-close-btn" id="movie-close-btn">
```
Comment (lojacho) [inline src/index.html]: As a suggestion, you can use different names for the class and the id, the code is going to be more clear.

**Review by lojacho — CHANGES_REQUESTED** [review]
Hello Belay,

Please check my suggestions, great job.

Thanks
Diego

**Review by Iqbal-Elham — APPROVED** [review]
## STATUS: APPROVED 🍾  🎆  🥇
You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖 
Congratulations! 🎉

**Pull request merged**